app/models/importers/eligible_fe_providers.rb

```diff
@@ -0,0 +1,55 @@
+require "csv"
+
+class Importers::EligibleFeProviders
+  include ActiveModel::Model
+
+  attr_reader :file, :academic_year
+
+  validate :validate_first_row_headers
+
+  def self.headers
+    %w[
+      ukprn
+      max_award_amount
+      lower_award_amount
```

Comment:
I think we only need the first figure. As I understand, the lower amount is always 50% of the higher amount.

We will also need to include additional fields such as contact email in this CSV to facilitate provider verification.

Reply:
> I think we only need the first figure. As I understand, the lower amount is always 50% of the higher amount.

I have removed `lower_award_amount` and can calculate it when we need it later

> We will also need to include additional fields such as contact email in this CSV to facilitate provider verification.

I don't have any requirements for that at this stage. are we able to kick out of scope for this PR? should be fairly trivial to add after though

Reply:
technically that's no problem. Let's check with the stakeholders as changing the CSV specification may cause confusion in testing for example.